cni/network/network.go

```diff
@@ -637,14 +638,12 @@ func (plugin *NetPlugin) createNetworkInternal(
 		NetNs:                         ipamAddConfig.args.Netns,
 		Options:                       ipamAddConfig.options,
 		DisableHairpinOnHostInterface: ipamAddConfig.nwCfg.DisableHairpinOnHostInterface,
-		IPV6Mode:                      ipamAddConfig.nwCfg.IPV6Mode,
+		IPV6Mode:                      ipamAddConfig.nwCfg.IPV6Mode, //TODO: check if this can be removed.
```

Comment:
nit: can you reword this to 'deprecated' pls? It would provide better context on if we can get rid of the whole mechanism around usage of IPV6Mode